src/ucp/rndv/rndv_am.c

```diff
@@ -77,9 +77,6 @@ ucp_proto_rndv_am_bcopy_complete(ucp_request_t *req)
     if (req->send.rndv.rkey != NULL) {
         ucp_proto_rndv_rkey_destroy(req);
     }
-    ucp_datatype_iter_mem_dereg(req->send.ep->worker->context,
-                                &req->send.state.dt_iter,
-                                UCS_BIT(UCP_DATATYPE_CONTIG));
```

Comment:
is it unrelated bug fix?

Reply:
This is a redundant memory deregistration since `bcopy` doesn't perform registration.
This didn't lead to an error for `CONTIG` datatype, but for `IOV`.